substrate/primitives/runtime/src/traits.rs

```diff
@@ -540,6 +540,9 @@ morph_types! {
 	/// Morpher to disregard the source value and replace with another.
 	pub type Replace<V: TypedGet> = |_| -> V::Type { V::get() };
 
+	/// Morpher to disregard the source value.
+	pub type Ignore = |_| -> () {};
+
```

Comment:
The naming is not really great and I don't think that we need this as a type here.

Reply:
Changed to `ReplaceWithDefault`.